tools/interactive/interactivetool_climate_notebook.xml

```diff
@@ -32,15 +33,15 @@
 
         #if $mode.mode_select == 'scratch':
             ## copy default notebook
-            cp '$__tool_directory__/default_notebook.ipynb' ./ipython_galaxy_notebook.ipynb &&
+            cp '/home/jovyan/default_notebook.ipynb' ./ipython_galaxy_notebook.ipynb &&
```

Comment:
should we ship the example notebook in the container, not in the tool?

Reply:
It is already in the docker container so I thought it would be best to have everything contained in the container. But I am probably wrong. Not sure what is best. What do you suggest?

Reply:
My initial idea was that people could create different tools with different default notebooks. But you probably have a point that if you create your own tool you also have your own container and therefore can ship a default notebook with it ...

I don't mind.